### PR TITLE
Slight tweak to Lens.root doc

### PR DESCRIPTION
**Pull request by sebastian**
I am not fully convinced about the wording here... couldn't really decide what would be most useful when I sat down to improve the wording. I hope the additional example would have helped former me.

### CODE REVIEW

File: lib/lens.ex [hunk 1]
```diff
@@ -17,12 +17,14 @@ defmodule Lens do
   end
 
   @doc ~S"""
-  Returns a lens that focuses on the whole data.
+  Returns a lens that focuses on all of the data in the given scope.
```
Comment (obrok) [inline lib/lens.ex]: I don't really like this "scope" - I don't think there is a mention of "scope" anywhere else in the doc. "all of the" vs. "the whole" is a good change, I think.

**Review by obrok — CHANGES_REQUESTED** [review]

File: lib/lens.ex (hunk 1 above)
Comment (sebastian) [reply lib/lens.ex]: But "all of the data" by itself is no less confusing. The problem which confused me was exactly that it wasn't _all of the data_ (which would have been the the `root` of the whole data structure), but rather the subtree that is currently focused on.

Do you have a suggestion for a better wording?

Comment (sebastian) [reply lib/lens.ex]: How about:

> Returns a lens that yields the entirety of the data currently under focus.

Comment (obrok) [reply lib/lens.ex]: > Returns a lens that yields the entirety of the data currently under focus.

This sounds good to me. Does it address the problem you have? You could also split it into 2 sentences, one saying the general "all the data" thing and the other describing the behaviour in a chain of lenses.

Comment (sebastian) [reply lib/lens.ex]: Yes, I think that would have solved my problem.

**Comment by sebastian** [comment]
Ready for re-review

**Pull request merged**